apps/web/src/vite-env.d.ts

```diff
@@ -1,2 +1,3 @@
 /// <reference types="vite/client" />
 /// <reference path="../../../global.d.ts" />
+/// <reference types="vite-plugin-svgr/client" />
```

Comment:
please add an empty line and make vscode do that for your (search for new line in the settings)